index.html

```diff
@@ -1,87 +1,108 @@
     <title>RedcXca</title>
-    <link rel="stylesheet" href="css/shared.css" />
-    <link rel="stylesheet" href="css/index.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet" />
-  </head>
+    <script src="https://code.jquery.com/jquery-3.7.1.min.js"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+        crossorigin="anonymous"></script>
+    <link href="./src/css/index.css" rel="stylesheet" />
+    <link href="./src/css/shared.css" rel="stylesheet" />
+    <style>
+        @import url("https://fonts.googleapis.com/css2?family=Dosis:wght@200..800&display=swap");
+    </style>
+</head>
 
-  <body>
-    <section class="hero" id="home">
-      <div class="title">
-        <img src="resources/images/早安.png" class="circle-image">
-        <h1>RedcXca</h1>
-        <p>Hi! I am a second year software engineering student at the University of Waterloo. I do digital art and play osu!</p>
-        <svg></svg><br>
-        <a class="commission-btn box-bold" href="https://docs.google.com/document/d/1CELGlKdLDq_-Usue3oQfcrawi-vKOXa3SOkzTHnqB5A/edit#heading=h.9lj1hkizyon2" target="_blank">Commission Info</a>
-        <div class="external-links"><br>
-          <ul>
-            <li><a id="pixiv" href="https://www.pixiv.net/en/users/49573484" target="_blank" onmouseover="ExternalLinksEnter('pixiv')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/pixiv.svg"><p>Pixiv</p></a></li>
-            <li><a id="instagram" href="https://www.instagram.com/RedcXca/" target="_blank" onmouseover="ExternalLinksEnter('instagram')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/instagram.svg"><p>Instagram</p></a></li>
-            <li><a id="twitter" href="https://twitter.com/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('twitter')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/twitter.svg"><p>Twitter</p></a></li>
-            <li><a id="osu" href="https://osu.ppy.sh/users/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('osu')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/osu.svg"><p>osu!</p></a></li>
-            <li><a id="github" href="https://github.com/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('github')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/github.svg"><p>GitHub</p></a></li>
-          </ul>
+<body>
+    <div class="hero"></div>
+    <nav id="navbar">
+        <div class="logo">RedcXca</div>
+        <div class="nav-link-wrapper">
+            <ul class="nav-links">
+                <li><a href="./index">About</a></li>
```

Comment:
these should have .html

Reply:
With a .htaccess file this won't really matter since its the same thing with or without .html suffix